doc/install/get-packages.rst

```diff
@@ -119,12 +119,9 @@ release::
 
 	echo deb https://download.ceph.com/debian-{release-name}/ $(lsb_release -sc) main | sudo tee /etc/apt/sources.list.d/ceph.list
 
-Ceph on ARM processors requires Google's memory profiling tools (``google-perftools``).
-The Ceph repository should have a copy at
-https://download.ceph.com/packages/google-perftools/debian. ::
-
-	echo deb https://download.ceph.com/packages/google-perftools/debian  $(lsb_release -sc) main | sudo tee /etc/apt/sources.list.d/google-perftools.list
+Ceph on ARM processors requires Google's memory profiling tools (``google-perftools``). ::
```

Comment:
as `libgoogle-perftools-dev` is already listed in d/control, i think we can just drop this paragraph.

Reply:
Hi @tchaikov, just clarifying, do you want me to take out the whole paragraph, even the ``Ceph on ARM processors requires Google's memory profiling tools (google-perftools).``?

Reply:
yes. as it's documented already.